gradle.properties

```diff
@@ -21,9 +21,11 @@ kotlin.code.style=official
 # resources declared in the library itself and none from the library's dependencies,
 # thereby reducing the size of the R class for that library
 android.nonTransitiveRClass=true
-
 # Implement CI (done by ourselves)
 org.gradle.daemon=true
 org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.configureondemand=true
+API_KEY="API_KEY"
+BASE_URL="BASE_URL"
+POSTER_URL="POSTER_URL"
```

Comment:
We don't need API_KEY and POSTER_URL, we will probably delete them

Reply:
Changed in next commit